### DISCUSSION TITLE
me.Color support for CanvasGradient

### DISCUSSION THREAD

**Discussion by parasyte** [melonjs/melonJS#1041] [Ideas]
This one will be tricky. :) WebGL might be a bit easier, since pixel shaders do the interpolation for you.

**Comment by obiot** [comment]
and by extension, a [`context.create*Gradient()`](https://html.spec.whatwg.org/multipage/scripting.html#fill-and-stroke-styles:canvasgradient-7) equivalent for the renderer API.
